src/iop/colorbalance.c

```diff
@@ -548,19 +1369,625 @@ static gboolean dt_iop_area_draw(GtkWidget *widget, cairo_t *cr, dt_iop_module_t
 }
 #endif
 
+static void autogrey_callback(GtkWidget *button, gpointer user_data)
+{
+  dt_iop_module_t *self = (dt_iop_module_t *)user_data;
+  if(self->dt->gui->reset) return;
+  
+  dt_iop_request_focus(self);
```

Comment:
I'm not sure those 2 calls are really necessary.

Reply:
Sorry I'm speaking of dt_iop_request_focus() and dt_control_queue_redraw();

Reply:
`dt_control_queue_redraw()` seems mandatory to draw the frame of the color picker